.github/ISSUE_TEMPLATE/create_release_branch.md

```diff
@@ -13,16 +13,16 @@ Make sure to follow the steps below and ensure all actions are completed and sig
 - **K8s version**: 1.xx
 
 <!-- Set this to the name of the person responsible for running the release tasks, e.g. @neoaggelos -->
-- **Owner**:
+- **Owner**: `who plans to do the work`
 
 <!-- Set this to the name of the team-member that will sign-off the tasks -->
-- **Reviewer**:
+- **Reviewer**:  @canonical/kubernetes
```

Comment:
should we keep this on a specific person?
If everyone is responsible, I guess no one feels responsible to tackle this.

Reply:
sure thing  👍🏻